src/net.h

```diff
@@ -723,7 +723,7 @@ class CNode
     {
         nRecvVersion = nVersionIn;
     }
-    int GetRecvVersion()
+    int GetRecvVersion() const
```

Comment:
Change seems fine to me, but why just this function? 
To avoid a PR per function might be a bit more thorough and do this for the entire class.
E.g. it looks like `GetRefCount` above, for example, is also missing const?